src/libgeds/TcpTransport.cpp

```diff
@@ -547,6 +548,7 @@ bool TcpPeer::processEndpointRecv(int sock) {
           auto message = "Error during GET_REPLY: " + std::to_string(ctx->hdr.error) +
                          "length: " + std::to_string(datalen) + " Ep: " + std::to_string(tep->sock);
           ctx->p->set_value(absl::AbortedError(message));
+          ctx->state = PROC_IDLE;
```

Comment:
makes sense. its just the peer which aborted the rpc. so we shall be ready for the next rpc